tox.ini

```diff
@@ -48,11 +48,11 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION RUNNING_IN_CI
 commands =
     make update_deps
     pip install --exists-action=w -r tests/ui/requirements.txt
-    pytest --driver Firefox -v tests/ui {posargs}
+    pytest -n 4 --boxed --driver Firefox -v tests/ui {posargs}
```

Comment:
```--boxed``` is used to that it destorys the db it creates each run. If not we would have to have a setup/teardown method, or we would have to have one setup that got the site as close to prod as prossible